{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
```

Comment:
this isn't what a pyproject.toml needs as a minimum requirement, we just need to adhere to the PEP requirements (see my PR on the core repo)

Reply:
Yup, it's necessary for poetry to use it though.